src/ansys/pre_commit_hooks/templates/AUTHORS.md

```diff
@@ -0,0 +1,12 @@
+This is the list of {{ project_name }}'s significant contributors.
+
+This file does not necessarily list everyone who has contributed code.
+
+For contributions made under a Corporate CLA, the organization is
+added to this file.
+
+If you have contributed to the repository and want to be added to this file,
+submit a request.
+
+
```

Comment:
Why not adding leading "#" chars in all these lines?

Reply:
Also, the file should be ``AUTHORS`` and not ``AUTHORS.md``

Reply:
I forgot to add the leading #